app/app.js

```
@@ -1449,32 +1458,32 @@ Vue.prototype.signMessage = async function (message) {
-                    message
-                })
-                result = '0x' + sig.payload.signature || ''
-                break
-            default:
-                break
+        case 'ledger':
+            await ensureLedgerEth(path)
+            const signature = await Vue.prototype.appEth.signPersonalMessage(
+                path,
+                Buffer.from(message).toString('hex')
+            )
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)